frontend/views/containers/proposals/ProposalTemplate.vue

```diff
@@ -100,6 +110,28 @@ export default {
       currentStep: 0
     }
   },
+  computed: {
+    ...mapGetters([
+      'groupShouldPropose'
+    ]),
+    footerMessage () {
+      const { value, total } = this.rule
+      const message = this.groupShouldPropose
+        ? 'According to your voting rules, <strong>{value} out of {total} members</strong> will have to agree with this.'
+        : 'Your group has less than 3 members, so <strong>this change will be immediate</strong> (no voting required).'
+
+      return this.L(message, { value, total })
```

Comment:
The parser won't be able to detect the string like this. Remember, the reason we **wrap** strings in `L` or `i18n` is so that the parser can detect and extract them. Doing it this way means the parser will not notice these strings, because it doesn't search for `L(variable)`, but `L('string')`